webapps/assembly/src/test/java/org/camunda/bpm/cockpit/plugin/base/ProcessDefinitionRestServiceTest.java

```diff
@@ -392,4 +393,73 @@ public void shouldSortResult() {
     verifyOrderedResult("incidents", "asc", Arrays.asList("p2", "B", "p1", "A"));
     verifyOrderedResult("incidents", "desc", Arrays.asList("A", "p1", "B", "p2"));
   }
+
+  @Test
+  public void shouldReturnCorrectStatistics() {
```

Comment:
**Context**: Test that covers the correct statistics calculation for process instances with incidents (different types)